### DISCUSSION TITLE
Need support for ppc64le architecture

### DISCUSSION THREAD

**Discussion by amintek** [prometheus-operator/prometheus-operator#4252] [Q&A] [answered]
hi guys, i have a Kubernetes cluster deployed on Linux/ppc64le platform, and i'm trying to figure out how to install prometheus-operator to monitor my k8s cluster. thanks in advance

**Comment by paulfantom** [comment]
We recently added support for s390x in https://github.com/prometheus-operator/prometheus-operator/pull/4351 and we welcome PRs extending this support to other platforms.

**Comment by paulfantom** [comment, marked as answer]
Plus I think ppc64le is already supported as seen on quay - https://quay.io/repository/prometheus-operator/prometheus-operator?tag=latest&tab=tags
